source/appModuleHandler.py

```diff
@@ -444,6 +454,52 @@ def _get_is64BitProcess(self):
 		self.is64BitProcess = not res
 		return self.is64BitProcess
 
+	def _get_isWindowsStoreApp(self):
+		"""Whether this process is a Windows Store (immersive) process.
+		An immersive process is a Windows app that runs inside a Windows Runtime (WinRT) container.
+		These include Windows store apps on Windows 8 and 8.1, and Universal Windows Platform (UWP) apps on Windows 10.
+		@rtype: bool
+		"""
+		if winVersion.winVersion.major and winVersion.winVersion.minor == (6, 1):

```

Comment:
This line is wrong. It should be:

`if (winVersion.winVersion.major, winVersion.winVersion.minor) == (6, 1):`

I prefer

`if (winVersion.winVersion.major, winVersion.winVersion.minor) < (6, 2):`